releasenotes/notes/1.1/prepare-1.1.0-732e167b70ab4d01.yaml

```diff
@@ -0,0 +1,38 @@
+---
+prelude: >
```

Comment:
do you think that maybe the improvements for MCX and MCPhase gates synthesis should also appear in the prelude?
see: https://github.com/Qiskit/qiskit/pull/11993

Reply:
Yup, thanks for that comment - I added this to the prelude! :)